README.md

```diff
@@ -3,3 +3,9 @@ Google Cloud Platform SDK command line utilities shell completion for fish
 
 ## Install
 Run install.sh to copy the files into your local fish config folder (~/.config/fish/).
+
+### Use Fisher
+
+```shell
+fisher add aliz-ai/google-cloud-sdk-fish-completion
```

Comment:
Should it be `fisher install aliz-ai/google-cloud-sdk-fish-completion`? I guess fisher released a new major in the meantime.

Reply:
Yes, this changed in Fisher [4.0.0](https://github.com/jorgebucaran/fisher/releases/tag/4.0.0).

```
❯ fisher install lgathy/google-cloud-sdk-fish-completion
fisher install version 4.3.1
Fetching https://codeload.github.com/lgathy/google-cloud-sdk-fish-completion/tar.gz/HEAD
Installing lgathy/google-cloud-sdk-fish-completion
           /var/home/howard/.config/fish/functions/gcloud_sdk_argcomplete.fish
           /var/home/howard/.config/fish/completions/gcloud.fish
           /var/home/howard/.config/fish/completions/gsutil.fish
Installed 1 plugin/s
```